src/components/pages/collectRecordFormPages/BenthicLitForm/BenthicLitTransectInputs.js

```diff
@@ -37,17 +37,11 @@ const BenthicLitTransectInputs = ({
 }) => {
   const { reefslopes, relativedepths, visibilities, currents, tides } = choices
 
-  const reefSlopeSelectOptions = [
-    ...getOptions(reefslopes.data),
-    { label: 'not reported', value: '' },
-  ]
-  const relativeDepthOptions = [
-    ...getOptions(relativedepths.data),
-    { label: 'not reported', value: '' },
-  ]
-  const visibilityOptions = [...getOptions(visibilities.data), { label: 'not reported', value: '' }]
-  const currentOptions = [...getOptions(currents.data), { label: 'not reported', value: '' }]
-  const tideOptions = [...getOptions(tides.data), { label: 'not reported', value: '' }]
+  const reefSlopeSelectOptions = getOptions(reefslopes.data)
```

Comment:
nitpick on naming - `reefSlopeOptions` (or if you want to keep `Select`, add it to the others too)